lib/livebook/evaluator/default_formatter.ex

```diff
@@ -80,4 +80,15 @@ defmodule Livebook.Evaluator.DefaultFormatter do
       reset: :reset
     ]
   end
+
+  defp error_type(error) do
+    cond do
+      mix_install_vm_error?(error) -> :mix_install_vm_error
```

Comment:
What if we keep the error generic? From the perspective of Livebook, all that matters is that this may require a runtime restart. So maybe :runtime_restart_required?

Reply:
We can change this at any point because the error output is internal to Livebook, but the alternative sounds good. I wasn't sure if we should show a more elaborate info for Mix.install, but the message is pretty clear already.